drill-common/src/commonMain/kotlin/com/epam/drill/common/PonyNameGenerator.kt

```diff
@@ -0,0 +1,71 @@
+package com.epam.drill.common
+
+val firstName = arrayOf(
+    "Crystal",
+    "Demure",
+    "Dissembling",
+    "Gorgeous",
+    "Hilarious",
+    "Sonorous",
+    "Sophisticated",
+    "Twinkling",
+    "Wonderful",
+    "Astonishing",
+    "Delightful",
+    "Lovely",
+    "Fancy",
+    "Shiny",
+    "Apple",
+    "Nectarine",
+    "Avocado",
+    "Orange",
+    "Apricot",
+    "Pear",
+    "Banana",
+    "Papaya",
+    "Pineapple",
+    "Peach",
+    "Grapefruit",
+    "Plum",
+    "Grapes",
+    "Persimmon",
+    "Kiwi",
+    "Pomegranate",
+    "Lime",
+    "Lemon",
+    "Quince",
+    "Mango",
+    "Tangerine",
+    "Melon"
+)
+
+val lastName = arrayOf(
+    "Drill",
+    "Delicacy",
+    "Destiny",
+    "Eloquence",
+    "Eternity",
+    "Felicity",
+    "Infinity",
+    "Insouciance",
+    "Languor",
+    "Lassitude",
+    "Nemesis",
+    "Oblivion",
+    "Passion",
+    "Serendipity",
+    "Solitude",
+    "Blossom",
+    "Bumblebee",
+    "Butterfly",
+    "Cynosure",
+    "Galaxy",
+    "Incandescence",
+    "Lullaby",
+    "Sunshine",
+    "Sweetheart",
+    "Syzygy",
+    "Elision"
+)
+
+fun generatePonyName() = "${firstName[(0 until firstName.size).random()]} ${lastName[(0 until lastName.size).random()]}"
```

Comment:
code duplication

Reply:
cuz what we need is generating names of equestria, not some czech republic
it's a long story

Reply:
Anyway, drill-common is not the right place for this thing
At the moment the only possible application for this is the coverage plugin, so it should be put there.

Reply:
no problem, will fix asap